set_env.sh

```diff
@@ -113,7 +123,7 @@ case $SIMPLE_PLATFORM in
         ;;
 
     windows)
-        PATH=/mingw64/bin:/usr/bin:/bin:$ROOT/Windows/System32:$ROOT/Windows
+        PATH=/mingw64/bin:/usr/bin:/bin:$ROOT/Windows/System32:$ROOT/Windows:$ROOT/PROGRA~1/Python37/Scripts/
```

Comment:
I am not sure the Windows VMs in elasticsearch-ci will have this version of Python, so if this is now required then it's an extra complication.  Previously we only needed Python to build PyTorch, which we don't do in CI.  Why do we need it now to build our own C++ code?

Reply:
Weirdly, this is the install location of cmake on the windows system I've been testing on. I've no idea why or how it's ended up under a python scripts folder. I'll need to investigate whether its possible to install it somewhere more sensible.